store/tikv/tests/snapshot_fail_test.go

```diff
@@ -206,7 +206,7 @@ func (s *testSnapshotFailSuite) TestRetryPointGetResolveTS(c *C) {
 
 	txn, err := s.store.Begin()
 	c.Assert(err, IsNil)
-	err = txn.Set([]byte("k1"), []byte("v1"))
+	txn.Set([]byte("k1"), []byte("v1"))
```

Comment:
Emm... I thought the linter should complain that the returned error is not used...
I think a better idea is to add an assertion